packages/twenty-front/src/modules/object-record/record-index/states/selectors/recordIndexAllRecordIdsComponentSelector.ts

```diff
@@ -0,0 +1,60 @@
+import { recordGroupIdsComponentState } from '@/object-record/record-group/states/recordGroupIdsComponentState';
+import { recordIndexRecordIdsByGroupComponentFamilyState } from '@/object-record/record-index/states/recordIndexRecordIdsByGroupComponentFamilyState';
+import { ObjectRecord } from '@/object-record/types/ObjectRecord';
+import { createComponentSelectorV2 } from '@/ui/utilities/state/component-state/utils/createComponentSelectorV2';
+import { ViewComponentInstanceContext } from '@/views/states/contexts/ViewComponentInstanceContext';
+import { v4 } from 'uuid';
+
+/**
+ * Do not use this key outside of this file.
+ * This is a temporary key to store the record ids for the default record group.
+ */
+const defaultFamilyKey = v4();
```

Comment:
why not hard code it?

Reply:
Ooops, I put an uuid first and change it to an hardcoded id but I had some issues with my rebase, I'll change it !